addons/note/models/note.py

```diff
@@ -52,10 +52,7 @@ def _get_default_stage_id(self):
     date_done = fields.Date('Date done')
     color = fields.Integer(string='Color Index')
     tag_ids = fields.Many2many('note.tag', 'note_tags_rel', 'note_id', 'tag_id', string='Tags')
-    message_partner_ids = fields.Many2many(
-        comodel_name='res.partner', string='Followers (Partners)',
-        compute='_get_followers', search='_search_follower_partners',
-        compute_sudo=True)
+    message_partner_ids = fields.Many2many(compute_sudo=True)
```

Comment:
Could leave a small comment mentioning it's adding that property to a mail.thread field maybe?
It confused me for a second :D